wordcloud_test.py

```diff
@@ -2,19 +2,18 @@
 from wordcloud import main
 from visual_wordcloud import VisualWordCloud
 
-TEXT = """This is a test test string string string."""
+# TEXT = "This is a test test string string string."
+TEXT = "initial_text.txt"
 TEST_WORD_TO_OCCURRENCE = {"string": 3, "test": 2, "this": 1, "is": 1, "a": 1}
 
 
 def test_word_cloud():
 
-    # TODO: Should word_to_occurrence be renamed to word_to_count ?
```

Comment:
As a courtesy, try not to remove other developer's TODOs 😉

Reply:
Oops!  My apologies.  I actually thought that it was a note for me.  I won't do it again.

Reply:
I'm sorry @EricaFinlay, I can see how this TODO looks like a reminder for you, and you're right it should be removed because you resolved the issue. I shouldn't have told you not to remove it 😢. It's still good to be careful about removing TODOs created by other people though.